src/plugins/intel_cpu/src/infer_request.h

```diff
@@ -4,106 +4,94 @@
+
+    void infer() override;
 
-    void InferImpl() override;
+    std::vector<ov::ProfilingInfo> get_profiling_info() const override;
 
-    std::map<std::string, InferenceEngine::InferenceEngineProfileInfo> GetPerformanceCounts() const override;
+    std::vector<std::shared_ptr<ov::IVariableState>> query_state() const override;
 
-    std::vector<std::shared_ptr<InferenceEngine::IVariableStateInternal>> QueryState() override;
+    void set_tensor(const ov::Output<const ov::Node>& port, const ov::Tensor& tensor) override;
+
+    void set_tensors_impl(const ov::Output<const ov::Node> port, const std::vector<ov::Tensor>& tensors) override;
+
+    ov::Tensor get_tensor(const ov::Output<const ov::Node>& port) const override;
+    std::vector<ov::Tensor> get_tensors(const ov::Output<const ov::Node>& _port) const override;
 
     /**
      * @brief      Sets the pointer to asynchronous inference request that holds this request
      * @param[in]  asyncRequest Pointer to asynchronous inference request
      */
-    void SetAsyncRequest(AsyncInferRequest* asyncRequest);
+    void set_async_request(AsyncInferRequest* asyncRequest);
 
     /**
-     * @brief If `_asyncRequest` is initialized throw exception with `InferenceEngine::INFER_CANCELLED` status if inference request is canceled
+     * @brief If `_asyncRequest` is initialized throw exception with `ov::Cancelled` status if inference request is
+     * canceled
      */
-    void ThrowIfCanceled() const;
+    void throw_if_canceled() const;
 
 protected:
-    InferRequestBase(InferenceEngine::InputsDataMap networkInputs,
-                     InferenceEngine::OutputsDataMap networkOutputs,
-                     std::shared_ptr<ExecNetwork> execNetwork_)
-    : IInferRequestInternal(networkInputs, networkOutputs), execNetwork(execNetwork_) {}
-
-    InferRequestBase(const std::vector<std::shared_ptr<const ov::Node>>& inputs,
-                     const std::vector<std::shared_ptr<const ov::Node>>& outputs,
-                     std::shared_ptr<ExecNetwork> execNetwork_)
-    : IInferRequestInternal(inputs, outputs), execNetwork(execNetwork_) {}
+    void create_infer_request();
+    InferenceEngine::Precision normToInputSupportedPrec(const std::pair<const std::string, ov::Tensor>& input) const;
+    void pushInput(const std::string& inputName, ov::Tensor& inputBlob, InferenceEngine::Precision dataType);
 
-    void CreateInferRequest();
-    InferenceEngine::Precision normToInputSupportedPrec(const std::pair<const std::string, InferenceEngine::Blob::Ptr>& input) const;
-    void pushInput(const std::string& inputName, InferenceEngine::Blob::Ptr& inputBlob, InferenceEngine::Precision dataType);
-
-    virtual void initBlobs() = 0;
-    virtual void PushInputData() = 0;
+    void init_tensor(const std::string& name);
+    void PushInputData();
 
     Graph* graph = nullptr;
-    std::unordered_map<std::string, void*> externalPtr;
+    mutable std::unordered_map<std::string, void*> _external_ptr;
```

Comment:
Please remove prefixes "_". We don't use such style in the plugin.

Reply:
Do we have any code style in cpu plugin, such as snake_case?

Reply:
Please just leave `external_ptr`

Reply:
ok, no problem